src/jukebox/components/player/core/__init__.py

```diff
@@ -29,12 +31,23 @@ class PlayerCtrl:
     def __init__(self):
         self._backends: Dict[str, Any] = {}
         self._active = None
+        self.player_status = None
+        self.status_poll_interval = 0.25
+        self.status_thread = multitimer.GenericEndlessTimerClass('player.timer_status',
```

Comment:
I like this! The Timer Classes are pretty well done and can be used for such use cases nicely!